### ISSUE TITLE
All Agents States Subscription In Node Agent SDK

### ISSUE THREAD

**Issue by danishmarif** [LivePersonInc/node-agent-sdk#88]
Is there any function in node agent sdk through which we can subscribe to all agents states and get update events/notifications once there state changes? 
Such as the RetrieveAvailableAgents REST API Call in LivePerson's Chat Agent API, which for the first and other call without a timestamp provide all agents states, and on API call along with the timestamp gives only the delta from that time.
There is a method, "subscribeAgentsState" , but according to the documentation I found and understanding of it, it'll update only the bot's state.

**Comment by bzier** [comment]
You are correct in your understanding of `subscribeAgentsState`. It only supports subscribing to the current agent. There is not currently any API for all agents.
